docs/formats/open_images_user_manual.md

```diff
@@ -89,6 +110,36 @@ Open Images dataset directory should have the following structure:
 To use per-subset image description files instead of `image_ids_and_rotation.csv`,
 place them in the `annotations` subdirectory.
 
+### Speeding up bounding box loading
+
+To load bounding box annotations,
+Datumaro needs to know the sizes of the corresponding images.
+By default, it will determine these sizes by loading each image from disk,
+which will make the loading process slow.
+
+You can speed up this process
+by extracting the image size information in advance
+and recording it in an `images.meta` file.
+This file must be located in the `annotations` directory,
+and must contain one line per image, with the following structure:
```

Comment:
```suggestion
If images are not available locally or you want to speed up the process,
you can provide an `images.meta` file.
Such file must be located in the `annotations` directory,
and must contain one line per image, with the following structure:
```

Reply:
I think just off-handedly mentioning the possibility that the images might not be available doesn't connect well with the other paragraphs, so I rewrite this whole section to incorporate this scenario.